pkgs/tools/networking/network-manager/applet.nix

```diff
@@ -20,8 +20,18 @@ in stdenv.mkDerivation rec {
       url = https://gitlab.gnome.org/GNOME/network-manager-applet/merge_requests/12.patch;
       sha256 = "0q5qbjpbrfvhqsprnwjwz4c42nly59cgnbn41w2zlxvqf29gjvwk";
     })
+    (fetchpatch {
+      # https://gitlab.gnome.org/GNOME/network-manager-applet/issues/11
+      # should be fixed in 1.8.15
+      name = "show-import-file-chooser.patch";
+      url = https://gitlab.gnome.org/GNOME/network-manager-applet/merge_requests/15.patch;
```

Comment:
We probably should not use unmerged MR for patch, as it can change. I would use the two specific commits (or [commit range](https://gitlab.gnome.org/GNOME/network-manager-applet/compare/280cb266099374e7564f0ef9ba18d86df0544d98...8c90a457c426e567c54c78ac023effcaea2d48a6) if it is available as a patch).

Reply:
I added all three commits separately, as the bluetooth one also shouldn't hurt and is included in the upstream MR.